sql/load_sources.sql

```diff
@@ -0,0 +1,31 @@
+DROP TABLE IF EXISTS pluto;
+DROP INDEX IF EXISTS pluto_geom_idx;
+CREATE TABLE IF NOT EXISTS "pluto" (
+	"bbl" text PRIMARY KEY NOT NULL,
+	"borough" char(2) NOT NULL,
+	"block" text NOT NULL,
+	"lot" text NOT NULL,
+	"address" text,
+	"land_use" char(2),
+	"wkt" geometry
+);
+CREATE INDEX pluto_geom_idx
+  ON pluto
+  USING GIST (wkt);
+
+\COPY pluto ("wkt", "borough", "block", "lot", "address", "land_use", "bbl") FROM './pluto.csv' DELIMITER ',' CSV HEADER;
```

Comment:
Because this is populating the pluto table just created, can we remove the creation and import of pluto in `create_tables.sql` and `import_tables.sql`? Just to not have duplicate sql

Reply:
done!